specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/examples/ApplicationGatewayAvailableSslOptionsGet.json

```diff
@@ -4,6 +4,142 @@
     "subscriptionId": "subid"
   },
   "responses": {
-    "200": {}
+    "200": {
+      "body": {
+        "name": "default",
+        "id": "/subscriptions/subid/resourceGroups//providers/Microsoft.Network/ApplicationGatewayAvailableSslOptions/default",
```

Comment:
Think the `resourceGroup//` needs to be removed from these ids. Another 16 hits below
```suggestion
        "id": "/subscriptions/subid/providers/Microsoft.Network/ApplicationGatewayAvailableSslOptions/default",
```

Reply:
That's what I got from the server
![image](https://user-images.githubusercontent.com/8080186/61275936-ade79580-a7b7-11e9-93f8-a397aa2e681b.png)

Reply:
Odd, the url in the spec is `/subscriptions/{subscriptionId}/providers/Microsoft.Network/applicationGatewayAvailableSslOptions/default"`

Reply:
Possible error on server's side. I'll try to follow up with AppGw team